phase3/weave-net/weave-net-daemon-set.json

```diff
@@ -1,133 +1,163 @@
-                                "name": "cni-conf",
-                                "mountPath": "/host/etc"
-                            },
-                            {
-                                "name": "dbus",
-                                "mountPath": "/host/var/lib/dbus"
-                            },
-                            {
-                                "name": "lib-modules",
-                                "mountPath": "/lib/modules"
-                            }
-                        ],
-                        "resources": {
-                            "requests": {
-                                "cpu": "10m"
-                            }
-                        }
-                    },
-                    {
-                        "name": "weave-npc",
-                        "image": "weaveworks/weave-npc:1.9.4",
-                        "resources": {
-                            "requests": {
-                                "cpu": "10m"
-                            }
-                        },
-                        "securityContext": {
-                            "privileged": true
-                        }
-                    }
-                ],
-                "restartPolicy": "Always",
-                "tolerations": [
-                    {
-                        "key": "node-role.kubernetes.io/master",
-                        "effect": "NoSchedule"
-                    }
-                ],
-                "serviceAccountName": "weave-net",
-                "securityContext": {
-                    "seLinuxOptions": {
-                        "type": "spc_t"
-                    }
-                },
-                "volumes": [
-                    {
-                        "name": "weavedb",
-                        "emptyDir": {}
-                    },
-                    {
-                        "name": "cni-bin",
-                        "hostPath": {
-                            "path": "/opt"
-                        }
-                    },
-                    {
-                        "name": "cni-bin2",
-                        "hostPath": {
-                            "path": "/home"
-                        }
-                    },
-                    {
-                        "name": "cni-conf",
-                        "hostPath": {
-                            "path": "/etc"
-                        }
-                    },
-                    {
-                        "name": "dbus",
-                        "hostPath": {
-                            "path": "/var/lib/dbus"
-                        }
-                    },
-                    {
-                        "name": "lib-modules",
-                        "hostPath": {
-                            "path": "/lib/modules"
-                        }
-                    }
-                ]
-            }
-        }
-    }
-}
+	"apiVersion": "extensions/v1beta1",
```

Comment:
apps/v1beta2 now...

Reply:
this commit is just reformatting ... i am not adding or removing anything here